dasharo-compatibility/basic-platform-setup.robot

```diff
@@ -0,0 +1,237 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=30 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+# Library    ../osfv-scripts/osfv_cli/src/osfv/rf/rte_robot.py
+# TODO: maybe have a single file to include if we need to include the same
+# stuff in all test cases
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../variables.robot
+Resource            ../keywords.robot
+Resource            ../keys.robot
+Resource            ../pikvm-rest-api/pikvm_comm.robot
+
+# TODO:
+# - document which setup/teardown keywords to use and what are they doing
+# - go threough them and make sure they are doing what the name suggest (not
+# exactly the case right now)
+Suite Setup         Run Keyword
+...                     Prepare Test Suite
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Test Cases ***
+BPS001.001 Power Control - Power On and Serial output
+    [Documentation]    Verifies if the DUT can be turned On and if the serial output can be read.
+    Power On
+    FOR    ${i}    IN RANGE    120
+        ${out}=    Read From Terminal
+        ${var}=    Check If Empty    ${out}
+        IF    ${var} == ${FALSE}    BREAK
+        Sleep    1s
+    END
+    IF    ${var}    Fail    Power On couldn't be detected
```

Comment:
Maybe more a matter of prerence, but https://robotframework.org/robotframework/latest/libraries/BuiltIn.html#Should%20Be%20True does the same , right?